MIDDLEWARE.md

```diff
@@ -0,0 +1,140 @@
+# Echo Logger Middleware
+
+## Overview
+
+The Logger middleware for Echo is a powerful tool designed to log HTTP requests and responses. It's essential for debugging, monitoring application performance, and tracking user interactions with your Echo-based web services.
+
+## Features
+
+- Customizable log format
+- Support for multiple output destinations
+- Ability to skip logging for specific routes
+- Customizable timestamp format
+- Option to hide certain information (e.g., IP addresses)
+
+## Installation
+
+The Logger middleware is included in the Echo framework. To use it, you need to import Echo and its middleware package:
+
+```go
+import (
+    "github.com/labstack/echo/v4"
+    "github.com/labstack/echo/v4/middleware"
+)
+```
+
+## Basic Usage
+
+To add the Logger middleware to your Echo instance with default settings:
+
+```go
+e := echo.New()
+e.Use(middleware.Logger())
+```
+
+This will log requests and responses to the console (stdout) using the default format.
+
+## Configuration
+
+The Logger middleware can be customized using `LoggerConfig`. Here's an example of how to use it with custom settings:
+
+```go
+e.Use(middleware.LoggerWithConfig(middleware.LoggerConfig{
+    Format: "${time_custom} ${status} ${method} ${host}${path} ${latency_human}\n",
+    CustomTimeFormat: "2006-01-02 15:04:05.00000",
```

Comment:
just picky and nothing important: I'd use some constant from [here](https://pkg.go.dev/time#pkg-constants) which can suggest the usage of these ones for the devs 😄